expected-consensus.md

```diff
@@ -171,60 +236,53 @@ if winning(electionProof) {
+New blocks (with multiple tickets) will have a few key properties:
 
-This new block (with multiple tickets) will have a few key properties:
+- All tickets in the `Tickets` array are signed by the same miner -- to avoid grinding through out-of-band collusion between miners exchanging tickets.
+- The `ElectionProof` was correctly generated from the ticket `K-|Tickets|-1` (with `|Tickets|` the length of the `Tickets` array) rounds back.
 
-- All tickets in are signed by the same miner -- to avoid grinding through out-of-band collusion between miners exchanging tickets.
-- The election proof was correctly generated from K rounds back, counting null blocks.
-
-```
-On looking back
-
-Here are a few examples to help illustrate how the randomness lookback parameter, 'K', functions, given the fact that it includes null blocks.
-
-At a given height 'N' (or for round N, which is functionally equivalent), sample your ticket from N-K rounds to generate your election proof.
-- This could mean that you are looking back J <= K actual blocks back: since we count null blocks, we may walk back multiple blocks (null blocks + actual block) within a single header.
-- This could mean you are sampling a null block ticket (i.e. some element in a 'Tickets' array) to generate your Election Proof.
-```
+This means that valid `ElectionProof`s can be generated from tickets in the middle of the `Tickets` array.
 
 ### Block Generation
 
-When you have a winning election proof and corresponding ticket, you may create a block. For more on this, see the [Mining spec](./mining.md#block-creation).
+Once a miner has a winning election proof generated over `i` rounds and `i` corresponding tickets, they may create a block. For more on this, see the [Mining spec](./mining.md#block-creation).
 
 ## Chain Selection
 
-As we saw, just as we can have no miners win in a round (leading to null block mining), multiple miners can be elected in a given round . This in turn means multiple blocks can be created in a round. In order to avoid wasting valid work done by miners, EC makes use of all valid blocks generated in a round
+Just as there can have 0 miners win in a round, multiple miners can be elected in a given round . This in turn means multiple blocks can be created in a round. In order to avoid wasting valid work done by miners, EC makes use of all valid blocks generated in a round
 
 ### Tipsets
 
-All valid blocks generated in a round form a 'Tipset' that participants will attempt to mine off of in the subsequent round (see above). Tipsets are valid so long as:
+All valid blocks generated in a round form a `TipSet` that participants will attempt to mine off of in the subsequent round (see above). TipSets are valid so long as:
 
-- All blocks in a Tipset must have the same parents
+- All blocks in a TipSet have the same parent TipSet
+- All blocks in a TipSet have the same number of tickets in their `Tickets` array
```

Comment:
@whyrusleeping @ZenGround0 spot-check on that extra condition. This is to ensure blocks at a given height were generated in the same number of rounds.

Reply:
this is to say, they have the same 'round' correct?

Reply:
yes. a ticket is produced at each round. invariant. will explicit.